python/apps/taiga/tests/integration/taiga/workflows/test_api.py

```diff
@@ -198,12 +223,51 @@ async def test_reorder_statuses_with_reorder_ok(client):
     assert res["statuses"] == [wf_status.b64id]
 
 
+async def test_reorder_statuses_404_not_found_pj_b64id(client):
+    pj = await f.create_project()
+    workflow = await sync_to_async(pj.workflows.first)()
+    wf_status = await sync_to_async(workflow.statuses.first)()
+    reorder_status = await sync_to_async(workflow.statuses.last)()
+
+    data = {"statuses": [wf_status.b64id], "reorder": {"place": "before", "status": reorder_status.b64id}}
+    client.login(pj.created_by)
+    response = client.post(f"/projects/{NOT_EXISTING_B64ID}/workflows/main/statuses/reorder", json=data)
+
+    assert response.status_code == status.HTTP_404_NOT_FOUND, response.text
+
+
+async def test_reorder_statuses_404_not_found_workflow_slug(client):
+    pj = await f.create_project()
+    workflow = await sync_to_async(pj.workflows.first)()
+    wf_status = await sync_to_async(workflow.statuses.first)()
+    reorder_status = await sync_to_async(workflow.statuses.last)()
+
+    data = {"statuses": [wf_status.b64id], "reorder": {"place": "before", "status": reorder_status.b64id}}
+    client.login(pj.created_by)
+    response = client.post(f"/projects/{pj.b64id}/workflows/{NOT_EXISTING_SLUG}/statuses/reorder", json=data)
+
+    assert response.status_code == status.HTTP_404_NOT_FOUND, response.text
+
+
+async def test_reorder_statuses_422_unprocessable_pj_b64id(client):
+    pj = await f.create_project()
+    workflow = await sync_to_async(pj.workflows.first)()
+    wf_status = await sync_to_async(workflow.statuses.first)()
+    reorder_status = await sync_to_async(workflow.statuses.last)()
+
+    data = {"statuses": [wf_status.b64id], "reorder": {"place": "before", "status": reorder_status.b64id}}
+    client.login(pj.created_by)
+    response = client.post(f"/projects/{INVALID_B64ID}/workflows/main/statuses/reorder", json=data)
+
+    assert response.status_code == status.HTTP_422_UNPROCESSABLE_ENTITY, response.text
+
+
 ################################################################################
 # WorkflowStatus DELETE /projects/<pj_b64id>/workflows/<wf_slug>/statuses/<ws_slug>
 ################################################################################
 
 
-async def test_200_delete_workflow_status_ok(client):
+async def test_delete_workflow_status_200_ok(client):
```

Comment:
This test returns status.HTTP_204_NO_CONTENT

Reply:
True, I'll change it.